src/tedpy/ted6000.py

```diff
@@ -29,19 +30,31 @@ def __init__(self, host: str, async_client: httpx.AsyncClient = None) -> None:
 
         self.endpoint_settings_results: Any = None
         self.endpoint_dashboard_results: Any = None
+        self.endpoint_mtu1_dashboard_results: Any = None
+        self.endpoint_mtu2_dashboard_results: Any = None
+        self.endpoint_mtu3_dashboard_results: Any = None
+        self.endpoint_mtu4_dashboard_results: Any = None
         self.endpoint_mtu_results: Any = None
         self.endpoint_spyder_results: Any = None
 
     async def update(self) -> None:
-        """Fetch data from the endpoints."""
+        """Fetch settings data from the endpoints."""
         await asyncio.gather(
             self._update_endpoint("endpoint_settings_results", ENDPOINT_URL_SETTINGS),
+        )
+
+        self._parse_mtus()
+
+        """Fetch MTU/Spyder data results"""
+        await asyncio.gather(
             self._update_endpoint("endpoint_dashboard_results", ENDPOINT_URL_DASHBOARD),
             self._update_endpoint("endpoint_mtu_results", ENDPOINT_URL_MTU),
             self._update_endpoint("endpoint_spyder_results", ENDPOINT_URL_SPYDER),
+            *map(lambda mtu: self._update_endpoint(
+                "endpoint_mtu%d_dashboard_results" % mtu.position, 
+                ENDPOINT_URL_MTUDASHBOARD, str(mtu.position)), self.mtus)
         )
 
-        self._parse_mtus()
         self._parse_spyders()
```

Comment:
For consistency I would move `_parse_spyders` up to where `_parse_mtus` is now.

Reply:
Resolved /w ddc5652